README.md

```diff
@@ -27,40 +44,42 @@ comfort and speed.
 
 All new backend code and React components should be well tested.
 
-## Supported operating systems
+## ![Supported systems](./app/assets/images/ui_components/system_settings.svg) **Supported operating systems**
```

Comment:
Is there anyway to not link to the SVG itself and still display it? If not, that's fine but it's kind of a weird experience when you click it. If you can avoid linking to it, please change it on all of the SVGs.